artifact/v3/manifest/manifest.cpp

```diff
@@ -84,14 +84,25 @@ ExpectedManifest Parse(mender::common::io::Reader &reader) {
 
 	auto str = reader.GetStream();
 	string line {};
+
+	// auto start_pos = str.get()->tellg();
```

Comment:
Remove?

Reply:
Removed now - this was part of the conversation below.